mmv1/products/backupdr/BackupPlan.yaml

```diff
@@ -0,0 +1,209 @@
+                  - FRIDAY
+                  - SATURDAY
+            - name: 'daysOfMonth'
+              type: 'Array'
+              description: 'Specifies days of months like 1, 5, or 14 on which jobs will run.'
+              item_type:
+                type: Integer
+            - name: 'weekDayOfMonth'
+              type: NestedObject
+              description: 'Specifies a week day of the month like FIRST SUNDAY or LAST MONDAY, on which jobs will run.'
+              properties:
+                - name: 'weekOfMonth'
+                  type: Enum
+                  description: 'WeekOfMonth enumerates possible weeks in the month, e.g. the first, third, or last week of the month.'
+                  enum_values:
+                    - WEEK_OF_MONTH_UNSPECIFIED
+                    - FIRST
+                    - SECOND
+                    - THIRD
+                    - FOURTH
+                    - LAST
+                  required: true
+                - name: 'dayOfWeek'
+                  type: Enum
+                  description: 'Specifies the day of the week.'
+                  enum_values:
+                    - DAY_OF_WEEK_UNSPECIFIED
+                    - MONDAY
+                    - TUESDAY
+                    - WEDNESDAY
+                    - THURSDAY
+                    - FRIDAY
+                    - SATURDAY
+                    - SUNDAY
+                  required: true
+            - name: 'months'
+              type: Array
+              description: 'Specifies values of months'
+              item_type:
+                type: Enum
+                name: 'MonthEnum'
+                description: 'Specifies the months of the year, like `FEBRUARY` and/or `MAY`, on which jobs will run.'
+                enum_values:
+                  - MONTH_UNSPECIFIED
+                  - JANUARY
+                  - FEBRUARY
+                  - MARCH
+                  - APRIL
+                  - MAY
+                  - JUNE
+                  - JULY
+                  - AUGUST
+                  - SEPTEMBER
+                  - OCTOBER
+                  - NOVEMBER
+                  - DECEMBER
+            - name: timeZone
+              type: String
+              description: The time zone to be used when interpreting the schedule.
+              required: true
+            - name: 'backupWindow'
+              type: NestedObject
+              description: |
+                A BackupWindow defines the window of the day during which backup jobs will run. Jobs are queued at the beginning of the window and will be marked as
+                `NOT_RUN` if they do not start by the end of the window.
+              properties:
+                - name: 'startHourOfDay'
+                  type: Integer
+                  description: |
+                    The hour of the day (0-23) when the window starts, for example, if the value of the start hour of the day is 6, that means the backup window starts at 6:00.
```

Comment:
Since it looks like 0 is a valid option, you may need `send_empty_value: true` here